app/models/asset_field_type.rb

```diff
@@ -74,6 +75,16 @@ def allowed_content_types
     end
   end
 
+  def asset_extension
+    unless existing_data['asset']['content_type'] == asset_content_type
+     errors.add(:asset, "Asset must be the same type of: #{existing_data['asset']['content_type']}")
+    end
+  end
+
+  def asset_field_type_id
```

Comment:
Ok sounds good, but this branch wont work on it's own without `asset_field_type_id`, but it will when merged with the Media Title story